--- gn-0.20161011/tools/gn/bootstrap/bootstrap.py.omv~	2016-10-11 16:10:45.108448295 +0200
+++ gn-0.20161011/tools/gn/bootstrap/bootstrap.py	2016-10-11 16:10:49.014417000 +0200
@@ -535,7 +535,6 @@ def write_gn_ninja(path, root_gen_dir, o
         'base/sys_info_posix.cc',
         'base/threading/platform_thread_internal_posix.cc',
         'base/threading/platform_thread_posix.cc',
-        'base/threading/thread_local_posix.cc',
         'base/threading/thread_local_storage_posix.cc',
         'base/threading/worker_pool_posix.cc',
         'base/time/time_posix.cc',
